spec/ciFullConf.js

```diff
@@ -31,7 +31,7 @@ exports.config = {
     'tunnel-identifier': process.env.TRAVIS_JOB_NUMBER,
     'build': process.env.TRAVIS_BUILD_NUMBER,
     'name': 'Protractor suite tests',
-    'version': '47',
```

Comment:
Saucelabs doesn't have FF 51 deployed yet, and last I checked we had issues with 50. I'll give it another look soon.